### DISCUSSION TITLE
Version 9.1 disable Multi-Language Support

### DISCUSSION THREAD

**Discussion by Uncreative-Dev** [vhessam/hessam-cms#26] [Q&A] [answered]
Hello,

first of all tahnk you for this package it made my life much easier. But i have a question. Is it possible to disable Multi-Language Support in the neweset version? The plan is to have the blog home page be on the /blog route not /{locale}/blog route. Just changing the routes didn't work. I haven't used Laravel that much yet so I would really appreciate the help.

Thanks

RagingRattlesnake

**Comment by samberrry** [comment]
Hello friend,
We have a plan to handle this, it will be handled with another branch or with a as an option. I've added this to our backlog. We will update you soon to handle single language. 

Thanks for your consideration.
Sam

**Comment by samberrry** [comment]
@RagingRattlesnake  Hi friend,
now for single language version of the package you can use this version:
`composer require binshops/laravel-blog:v8.1.1`

To see installation instructions go to Readme file - single language version sections.

**Comment by samberrry** [comment, marked as answer]
https://github.com/binshops/laravel-blog/discussions/90
